src/query/numeric_opfunc.c

```diff
@@ -41,6 +41,10 @@
 #include "byte_order.h"
 #include "object_primitive.h"
 
+#if defined (__cplusplus)
+using namespace std;		/* for fpclassify */
```

Comment:
is this needed? if true, we should rather use std:: instead of using namespace std.